### PR TITLE
Improved README and added screenshot.

### LINKED ISSUE #1: Convert README to README.md and add screenshot

### PR DESCRIPTION
**Pull request by R4H33M**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Original Author: Jón Ragnarsson  
+Other original contributors: Michael Pfeiffer
+
+![8Dock screenshot](8dock.png)
+
+8Dock is a desktop dock.   
+Drag files and applications into 8Dock and double-click to launch them.   
+Right-click to switch between big and small icons.
+
+<ins>History:</ins>
```
Comment (humdingerb) [inline README.md]: Can you put the whole history into a "Changelog" file?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## 8Dock v. 0.9.5
```
Comment (humdingerb) [inline README.md]: Remove the version part. Then we don't have to keep updating the Readme.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Original Author: Jón Ragnarsson  
+Other original contributors: Michael Pfeiffer
```
Comment (humdingerb) [inline README.md]: Let's remove poor Michael and stick him with the rest into the copyrights, wehen we have a package built via haikuporter.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+![8Dock screenshot](8dock.png)
+
+8Dock is a desktop dock.   
+Drag files and applications into 8Dock and double-click to launch them.   
+Right-click to switch between big and small icons.
+
```
Comment (scottmc) [inline README.md:9]: You can drop everything from History as it's rather old by now.

**Review by humdingerb — CHANGES_REQUESTED** [review]
The screenshot isn't neatly cropped. There's some background at the left and, I think, a line missing from the window at the bottom.
Also, can you take a more realistic and nicer looking dock? Maybe a height of 6 slots, with the first 4 filled with 2 apps, 1 file, 1 folder?

**Comment by R4H33M** [comment]
I've made the requested changes.

**Comment by humdingerb** [comment]
Thank you!

**Pull request merged**